AK/String.h

```diff
@@ -159,6 +159,8 @@ class String {
     using SearchDirection = StringUtils::SearchDirection;
```

Comment:
Commit message nitpick: It's "a helper", not "an helper".

Reply:
Ah ok was not sure and english is weird with `an` and `H`
Will fix shortly